scripts/localstack/init/localstack_init.sh

```diff
@@ -8,6 +8,16 @@ awslocal dynamodb create-table \
     AttributeName=id,KeyType=HASH \
   --provisioned-throughput ReadCapacityUnits=1000,WriteCapacityUnits=1000
 
+sleep 5
+
+awslocal dynamodb update-table \
+    --table-name identity-verify \
+    --attribute-definitions AttributeName=yotiSessionId,AttributeType=S \
+    --global-secondary-index-updates \
+        "[{\"Create\":{\"IndexName\": \"yotiSessionId-index\",\"KeySchema\":[{\"AttributeName\":\"yotiSessionId\",\"KeyType\":\"HASH\"}], \
+        \"ProvisionedThroughput\": {\"ReadCapacityUnits\": 20, \"WriteCapacityUnits\": 10 }, \
+        \"Projection\":{\"ProjectionType\":\"ALL\"}}}]"
```

Comment:
I've created [a PR to add this to the real infra](https://github.com/ministryofjustice/opg-sirius-infrastructure/pull/2196)